confidential-data-hub/kms/src/plugins/mod.rs

```diff
@@ -54,16 +54,25 @@ pub async fn new_decryptor(
 pub enum VaultProvider {
     #[strum(ascii_case_insensitive)]
     Kbs,
+
+    #[cfg(feature = "aliyun")]
+    #[strum(ascii_case_insensitive)]
+    Aliyun,
 }
 
 /// Create a new [`Getter`] by given provider name and [`ProviderSettings`]
 pub async fn new_getter(
     provider_name: &str,
-    _provider_settings: ProviderSettings,
+    provider_settings: ProviderSettings,
```

Comment:
please do not delete the underscore `_` here. If we do not have `#[cfg(feature = "aliyun")]` the lint will raise an error